### DISCUSSION TITLE
OrangePi 5 Plus NVME boot issues

### DISCUSSION THREAD

**Discussion by workinhd** [Joshua-Riek/ubuntu-rockchip#450] [General]
I can boot from NVME from a fresh image, but I swear as soon as I put it in the case, it stops working.  Could the case be pushing the MASKrom button?  I am not sure if that would cause it.  It does not appear to be shorting out, the case is metal.  Would the MASKrom button do that, or am I getting any update while it sits that causes the issue?  If it is in the case but the top isn't on, it may work for awhile.  It is the strangest thing.

**Comment by workinhd** [comment]
Will try this when I get home.  Maybe the orangepi-config/system/install SPI for NVME isn't working correctly.

sudo dd if=/lib/u-boot-orangepi-rk3588/rkspi_loader.img of=/dev/mtdblock0 conv=notrunc
sync

**Comment by workinhd** [comment]
Still having issues booting from NVME drive.  It usually happens after a power failure.  Then it will boot to a Initramfs screen and if I am lucky, i can power down the device with the button without disconnecting power, then hit the power button and it will boot perfectly.  Sometimes it takes multiples tries to get it to boot.  I know the problem is it can not see the nvme, but I do not know why sometimes it works perfectly.

**Reply by MarkhamLee** [reply]
What kind of power supply are you using? I was having this issue using one of those beefy multi-chargers that had no issues charging multiple MacBooks in addition to another similar device with a 40W USB C output. I then tried my official Apple MacBook charger and had no problems, went out and bought the official Orange Pi 5 charger and had no issues with that one as well. TL/DR: I had this issue, but not when using an official MacBook charger or the official Orange Pi 5 charger.

**Reply by workinhd** [reply]
I use the power supply that came with the orange pi 5 plus.  I do have a wifi 6 card, nvme drive, and fan installed.  I was thinking of unplugging the fan to see if that helps.
